// @moroo/wdio-slack-reporter/build/constants.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ERROR_MESSAGES = exports.EVENTS = exports.SLACK_REQUEST_TYPE = exports.EMOJI_SYMBOLS = exports.DEFAULT_INDENT = exports.FINISHED_COLOR = exports.DEFAULT_COLOR = exports.FAILED_COLOR = exports.SUCCESS_COLOR = exports.SLACK_ICON_URL = exports.SLACK_NAME = void 0;
exports.SLACK_NAME = 'WebdriverIO Reporter';
exports.SLACK_ICON_URL = 'https://webdriver.io/img/webdriverio.png';
exports.SUCCESS_COLOR = '#36a64f';
exports.FAILED_COLOR = '#dc3545';
exports.DEFAULT_COLOR = '#D3D3D3';
exports.FINISHED_COLOR = '#4366c7';
exports.DEFAULT_INDENT = '\t';
exports.EMOJI_SYMBOLS = {
    PASSED: ':white_check_mark:',
    SKIPPED: ':double_vertical_bar:',
    PENDING: ':grey_question:',
    FAILED: ':x:',
    ROKET: ':rocket:',
    CHECKERED_FLAG: ':checkered_flag:',
    STOPWATCH: ':stopwatch:',
};
exports.SLACK_REQUEST_TYPE = {
    WEB_API_POST_MESSAGE: 'web-api:message',
    WEB_API_UPLOAD: 'web-api:upload',
    WEBHOOK_SEND: 'webhook:send',
};
exports.EVENTS = {
    POST_MESSAGE: 'slackReporter:postMessage',
    UPLOAD: 'slackReporter:upload',
    SEND: 'slackReporter:send',
    RESULT: 'slackReporter:result',
    SCREENSHOT: 'slackReporter:screenshot',
};
exports.ERROR_MESSAGES = {
    UNDEFINED_SLACK_OPTION: 'Slack Option is undefined. Please Check Slack Option.',
    NOT_USING_WEBHOOK: 'Not using webhook.',
    NOT_USING_WEB_API: 'Not using web-api.',
    DISABLED_OPTIONS: 'Disabled notifyFailedCase or uploadScreenshotOfFailedCase options.',
};
